bin/src/pcap.rs

```diff
@@ -21,12 +21,15 @@ use tokio_util::io::ReaderStream;
 // pcap data to a channel that's piped to the client.
 pub async fn get_pcap(
     State(state): State<Arc<ServerState>>,
-    Path(qmdl_name): Path<String>,
+    Path(mut qmdl_name): Path<String>,
 ) -> Result<Response, (StatusCode, String)> {
     let qmdl_store = state.qmdl_store_lock.read().await;
+    if qmdl_name.ends_with("pcapng") {
+        qmdl_name = qmdl_name.trim_end_matches(".pcapng").to_string();
+    }
```

Comment:
i think instead of stripping the extension off here and in `get_qmdl`, we can let axum do this for us in `daemon.rs`:

```rust
    Router::new()
        .route("/api/pcap/{name}.pcap", get(get_pcap))
        .route("/api/qmdl/{name}.qmdl", get(get_qmdl))
```

then the name will be the stripped name automatically